source/includes/_endpoint-deck.md

```diff
@@ -0,0 +1,492 @@
+    "index": {
+        "https://beta.crunch.io/api/datasets/cc9161/decks/4fa25/": {
+          "name": "my new deck",
+          "creation_time": "1986-11-26T12:05:00",
+          "id": "4fa25",
+          "is_public": false,
+          "owner_id": "https://beta.crunch.io/api/users/abcd3/",
+          "owner_name": "Real Person"
+        },
+        "https://beta.crunch.io/api/datasets/cc9161/decks/2b53e/": {
+          "name": "Default deck",
+          "creation_time": "1987-10-15T11:45:00",
+          "id": "2b53e",
+          "is_public": true,
+          "owner_id": "https://beta.crunch.io/api/users/4cba5/",
+          "owner_name": "Other Person"
+        }
+    },
+    "order": "https://beta.crunch.io/api/datasets/223fd4/decks/order/"
+}
+```
+
+The decks catalog tuples contain the following keys:
+
+
+Name | Type | Description
+---- | ---- | -----------
+name | string | Human-friendly string identifier
+creation_time | timestamp | Time when this deck was created
+id | string | Global unique identifier for this deck
+is_public | boolean | Indicates whether this is a public deck or not
+owner_id | url | Points to the owner of this deck
+owner_name | string | Name of the owner of the deck (referred by `owner_id`)
+
+To determine if a deck belongs to the current user, check the `owner_id` attribute.
+
+#### POST
+
+POST a shoji:entity to create a new deck for this dataset. The only required body attribute is "name"; other attributes are optional.
+
+```json
+{
+    "element": "shoji:entity",
+    "self": "https://beta.crunch.io/api/datasets/223fd4/decks/",
+    "body": {
+        "name": "my new deck",
+        "description": "This deck will contain analyses for a variable",
+        "is_public": false
+    }
+}
+```
+
+```http
+HTTP/1.1 201 Created
+Location: https://beta.crunch.io/api/datasets/223fd4/decks/2b3c5e/
+
+```
+
+The `shoji:entity` POSTed accepts the following keys
+
+Name | Type | required | Description
+---- | ---- | -----------
+name | string | Yes | Human-friendly string identifier
+description | string | No | Optional longer string with additional notes
+is_public | boolean | No | If `true`, all users with view access to this dataset will be able to read and export this deck and its analyses; if `false`, the default value, the deck remains private for the current user only.
+
+
+#### PATCH
+
+It is possible to bulk-edit many decks at once by PATCHing a shoji:catalog to 
+the decks' catalog.
+
+```json
+{
+    "element": "shoji:catalog",
+    "index": {
+        "https://beta.crunch.io/api/datasets/cc9161/decks/4fa25/": {
+          "name": "Renamed deck",
+          "is_public": true,
+        }
+    },
+    "order": "https://beta.crunch.io/api/datasets/223fd4/decks/order/"
+}
+```
+
+The following attributes are editable on this endpoint:
+
+ * name
+ * description
+ * is_public
+ 
+The other attributes are read-only and will 400 if the payload tries to change
+them.
```

Comment:
We say something different below, in the PATCH entity discussion. Which is correct?